lib/loader/index.js

```diff
@@ -130,6 +142,41 @@ function instantiate(module, imports) {
     memory_free(ptr);
   }
 
+  /** Gets a function from the module's memory by its pointer, for JavaScript only. */
+  function getFunction(ptr) {
+    return table.get(ptr);
+  }
+
+  /** Gets a function with 0 parameters from the module's memory by its pointer. */
+  function getFunction0(ptr) {
```

Comment:
Are these overloads necessary? Users could as well do checks like these on their own, if they have a need for it.

Reply:
I think it's pretty useful because users don't then need to understand how wasm memory really works and where to find the functions inside it.
And all the getFunctionX reduce the boilerplate.
It's easier to get a callback with theses functions.

What do you think ?

Reply:
`getFunction` is fine, yeah, but I am not so sure about `getFunctionX` which seems like something that is likely to end up unused most of the time by most developers anyway while adding to package size.

Reply:
You're right, I will check a clever solution for doing this instead of `getFunction0`, `getFunction1`... May be only one which does all case ? But what about the typescript type ? What do you think of `getFunction(ptr: number, nbArg: number): (...params: any[]) => any` and calling with undefined/null will don't do the arguments checking ?